project_assets/logo.svg

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I was not sure if `assets` would be expected to refer to more than media materials. Maybe media would be a better name?